src/Tools/BuildValidator/Program.cs

```diff
@@ -278,25 +288,30 @@ private static bool ValidateFiles(IEnumerable<FileInfo> originalBinaries, Option
             try
             {
                 // Find the embedded pdb
-                using var originalBinaryStream = originalBinary.OpenRead();
+                using var originalBinaryStream = File.OpenRead(assemblyInfo.FilePath);
```

Comment:
nit: feels like this line doesn't need to change if the declaration of `originalBinary` is moved up.